source/_posts/2023-09-06-application-form-2023.md

```diff
@@ -0,0 +1,106 @@
+
+The application form will be **open until September 30, 2023**.
+
+<p style="text-align: center;">
+<a href="https://forms.gle/Ji7Vvj1y2bMf9pUz7" class="inline-block py-2 no-underline px-6 !text-white bg-[#7f52ff] rounded-3xl hover:bg-[rgba(127,82,255,.8)]">Apply</a>
+</p>
+
+Note: This is not a job for PHP developers or those who want to become one. The development of PHP is done in the С programming language, and requires proficiency in system programming.
+
+
+## What are the requirements?
+
+As before, we require previous experience in PHP core development. Ideally you should have all, but at least some of the following:
+
+* Pull-requests / commits to [php/php-src](https://github.com/php/php-src/)
+* Experience in writing PHP extensions
+* Participation in PHP mailing lists
+* Contributions to other open-source projects
+
+This time, however, we have a clearer idea of the tasks that can be engaged in and so the pool of considered candidates will be wider.
+
+
+## What am I going to work on?
+
+Strategic areas we’d like you to work on:
+
+* Security
+* Performance
+* Infrastructure
+* Web APIs (json, PDO, streams, etc.)
+* Documentation
+
+This is not an exhaustive list, and if you would like to do something else, make sure to mention it in the form. The main criteria is a pragmatic approach. In this sense, we adhere to the position that Rasmus&nbsp;Lerdorf, the author of PHP, voiced back in 2014:
+
+```Text
+And rather than piling on language features with the main justification
+being that other languages have them, I would love to see more focus on
+practical solutions to real problems. The password_hash() addition in
+5.5 is a good example of addressing a real Web problem.
+…
+I know these are much harder and less sexy than adding a couple
+of tokens to the yacc grammar, but when we start looking at a new major
+release this is what I hear from people who are running PHP in
+production environments are looking for. Language features are nice, if
+they are well thought out, but significant infrastructure improvements
+are nicer.
+```
+*[https://news-web.php.net/php.internals/71525](https://news-web.php.net/php.internals/71525)*
+
+
+## What does working for the PHP foundation look like?
+
+Working for the PHP Foundation offers a lot of freedom, but it also requires discipline. Your work will roughly consist of the following:
+
+* 40% Triaging issues and fixing bugs
+* 30% Working on strategic areas
+* 20% Code reviews
+* 10% Preparing RFCs and developing new language features
+
+As you can see, developing new RFCs is a fairly small part of the job. Most of the work is a typical maintenance of an open source project.
+
+We are a fully distributed team. You can work from any place* and at any time. All communication is asynchronous via GitHub, Slack, and Email. We meet for synchronous calls 1-2 times a month.
```

Comment:
there is a star missing here for formatting.

Reply:
This is a footnote remark about sanctioned territories. Should I format it differently?

Reply:
aah, i thought it was about making something bold. don't mind me then :D